pkg/agent/datamodel/sig_config.go

```diff
@@ -480,7 +484,7 @@ var (
 		ResourceGroup: AKSCBLMarinerResourceGroup,
 		Gallery:       AKSCBLMarinerGalleryName,
 		Definition:    "V1",
-		Version:       LinuxSIGImageVersion,
+		Version:       FrozenCBLMarinerV1SIGImageVersionForDeprecation,
```

Comment:
oh, I guess we never published v1gen2...

Reply:
yeah we didnt but we always built it lol